salt/modules/win_pkg.py

```diff
@@ -44,6 +44,16 @@ def __virtual__():
     return False
 
 
+def normalize_name(name):
+    '''
+    This function needed for the pkg state module
+    :param str name:
+    :return: name
+    :rtype str:
+    '''
+    return name
```

Comment:
@cachedout For some reason when I was testing it kept failing because it couldn't find normalize_name. Which makes sense because it's only defined in `module/yumpkg.py`. But for some reason it kept trying to load it. The line that was trying to load it was:
```
_normalize_name = __salt__.get('pkg.normalize_name', lambda pkgname: pkgname)
```
To me, it should have seen that `pkg.normalize_name` didn't exist and then use the lambda function instead... but for some reason, it kept trying to find `normalize_name`. I've had it do strange stuff like this before and then when I came in the next day everything worked. I'll test this morning and see if it's still trying to find `normalize_name`.